kfp/kfp_support_lib/src/kfp_support/workflow_support/utils/kfp_utils.py

```diff
@@ -89,10 +89,15 @@ def runtime_name(ray_name: str = "", run_id: str = "") -> str:
     def dict_to_req(d: dict[str, Any], executor: str = "transformer_launcher.py") -> str:
         res = f"python {executor} "
         for key, value in d.items():
-            if isinstance(value, str):
-                res += f'--{key}="{value}" '
+            if isinstance(value, bool):
+                if value:
+                    res += f"--{key} "
+            elif isinstance(value, str):
+                if value != "":
+                    res += f'--{key}="{value}" '
```

Comment:
what if the value has double-quotes in it?

Reply:
It shouldn't have double-quotes. For example, the dictionary keys and values should be quoted with single-quote.

Reply:
I suggest to use 
```python
if len(value) != 0:
   res += f'--{key}="{value}" '
```

Reply:
It shouldn't be `None` value here, because if the user keep the input as blank field it will be an empty string. Also `len(None)` will throw an exception.

Reply:
After thinking about it some more I think it is even simpler. Considering that the code has proper defaults for everything, I thing we can do this:

```
        for key, value in d.items():
            if str(value) != "":
                if isinstance(value, str):
                    res += f'--{key}="{value}" '
                else:
                    res += f"--{key}={value} "
        return res
```
So wear just removing parameters with empty value